### ISSUE TITLE
Tile is now white after updating to HASS 0.113.1

### ISSUE THREAD

**Issue by SLiK420** [JuanMTech/google_dark_theme#30]
Tile background is now white after updating to V 0.113.1. Is there a setting or something to change back?

![hassexmaple](https://user-images.githubusercontent.com/58960013/88597352-eb623e00-d01b-11ea-93bf-4625c3272cf0.JPG)

**Comment by davidrustingha** [comment]
Same here. Deleted cache and such, but no change..

**Comment by Edmond76** [comment]
Hello,

On my side, I've found a solution:
There was a "--paper-card-background-color" into my file "mini-media-player-bundle.js" (it comes from an old module "Mini Media Player")
I've replaced **--paper-card-background-color** by **--card-background-color** It works now! Black background is back again.

Search with Notepad++ for example, on your files if the text "--paper-card-background-color" exists somewhere and change it like bellow.

Ed

HassOS 4.11
Home Assistant 0.113.1
Google Dark Theme v1.7

**Comment by Edmond76** [comment]
Hello,

On my side, I've found a solution:
Before install HACS I used theme like this with a theme.yaml file:

Into configuration.yaml:

> frontend:
>   themes: !include themes.yaml

I change my config file like this:

> frontend:
>   themes: !include_dir_merge_named themes

After a quick reboot, now it works. Black (#202124) background is back!

Ok, I cannot use anymore my old themes but I prefer this Google Dark Theme and if I want to change, and can now add a new ones with HACS.

Just to say that the issue doesn't come from the Google Dark Theme but your config, probably your other themes who forced the "card-background-color" to another value.

Ed

HassOS 4.11
Home Assistant 0.113.1
Google Dark Theme v1.7
